src/no/nav/statistikkapi/kafka/DatavarehusKafkaProducerImpl.kt

```diff
@@ -11,7 +12,7 @@ interface DatavarehusKafkaProducer {
     fun send(kandidatutfall: Kandidatutfall)
 }
 
-class DatavarehusKafkaProducerImpl(config: Properties) : DatavarehusKafkaProducer {
+class DatavarehusKafkaProducerImpl(config: Properties, val repository: StillingRepository) : DatavarehusKafkaProducer {
```

Comment:
Hmm, er det riktig at en kafkaproducer skal ha ansvar for å lese i db? Lurer på om den burde være så "dum" at den bare sender meldinger som er ferdig bygd opp.